src/media/js/categories.js

```diff
@@ -3,23 +3,30 @@ define('categories', ['core/l10n'], function(l10n) {
     var gettext = l10n.gettext;
 
     return [
-        {slug: 'games', name: gettext('Games')},
```

Comment:
Don't change the order - it's important, it's the order in which they are displayed